sdk-uikits/javascript/getting-started/support.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Support"
+description: "Learn how to get support for Botstacks JavaScript SDK, including feature requests, bug fixes, and technical support from the team."
+icon: "circle-info"
+iconType: "solid"
+---
+
+<Warning> The JavaScript SDK is still in active development.</Warning>
+## GitHub
```

Comment:
Can you remove our comment out (<!-- content -->) for now. We can't track issues on github yet because the javascript sdk is not open-source

Reply:
Removed github section for issues tracking on the latest commit [e51d7d4](https://github.com/BotStacks/mintlify-docs/pull/2/commits/e51d7d4417c16dd71da65974edb0709c14c9e252)